Assets/Countly/Plugins/CountlySDK/Countly.cs

```diff
@@ -360,7 +359,7 @@ private void CreateListOfIBaseService()
             _listeners.Clear();
 
             _listeners.Add(_push);
-            _listeners.Add(Views);
+            _listeners.Add((ViewCountlyService) Views);
```

Comment:
out of curiosity, why this require casting but others not?

Reply:
It can't convert the interface without the explicit cast there. That's why it's required to do the cast